BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -131,6 +137,42 @@ public struct AccessibilityElement: Element {
     }
 }
 
+extension AccessibilityElement.Trait: Hashable, Equatable {
```

Comment:
Does "hashable" still mean something if `adjustable` now has closures? I guess this is more a hash of the type, rather then the actual content, right?

Reply:
Yeah, was thinking the same. Do we need `Hashable`/`Equatable` conformance on this?

Reply:
I think it's b/c we throw it into a `Set` and need to check traits in said set. But I think only the type of the trait matters, so it's OK that we don't care about the closures.

Reply:
Yep this is exactly what's going on.

Reply:
Added comment explaining the conformance.